tests/docker-stacks-foundation/test_packages.py

```diff
@@ -154,25 +143,26 @@ def _check_import_packages(
     Note: using a list of packages instead of a fixture for the list of packages
     since pytest prevents the use of multiple yields
     """
-    failures = {}
+    failed_imports = []
     LOGGER.info("Testing the import of packages ...")
     for package in packages_to_check:
         LOGGER.info(f"Trying to import {package}")
         try:
             check_function(package_helper, package)
         except AssertionError as err:
-            failures[package] = err
-    if len(failures) > 0:
-        raise AssertionError(failures)
+            failed_imports.append(package)
+            LOGGER.error(f"Failed to import package: {package}, output:\n  {err}")
```

Comment:
Nesting error message in a json makes it much worse when printing, simple `LOGGER.error` looks much better.
The downside is that there are several messages instead of one, but I actually prefer reading several beautiful messages than one unreadable json.